manifests/zero.pp

```diff
@@ -15,7 +15,8 @@
   $garbage_collection = false,
   $auto_gc = false,
   $auto_gc_timespan = '3 days ago',
-  $ignore_xdir_hardlinks = false
+  $ignore_xdir_hardlinks = false,
+  $creator_version = '2.1.19',
```

Comment:
It's a parameter defaulting to current value. Dan will override it when the time comes.